src/contracts/payloads/interfaces/IWithPayloadsManager.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IWithPayloadsManager {
+  /**
+   * @dev Emitted when the payload manager gets updated.
+   * @param oldPayloadsManager The address of the old payload manager.
+   * @param newPayloadsManager The address of the new payload manager.
+   */
+  event PayloadsManagerUpdated(address oldPayloadsManager, address newPayloadsManager);
```

Comment:
by design decision, we are not emitting old addresses when updating. just the new one